### ISSUE TITLE
Using PWD has unintended consequences

### ISSUE THREAD

**Issue by alexellis** [jsiebens/terraform-equinix-faasd#5]
Using PWD has unintended consequences

```
    export PWD=$(terraform output -raw basic_auth_password)
    export OPENFAAS_URL=https://derek.example.com
    echo $PWD | faas-cli login -u admin --password-stdin
```

This changes my prompt to the password :)

```
alex@alex-nuc8:~$ pwd
/home/alex
alex@alex-nuc8:~$ export PWD="SECRETPASSWORD"
alex@alex-nuc8:SECRETPASSWORD$ 
```

I would suggest `PASSWORD` or `PASSWD`?

**Comment by jsiebens** [comment]
fixed in a2bd52f5cf5e9088da7fa6a8f5cfe3eec2c90702

**Comment by alexellis** [comment]
`PASSWORD` is the string we use in arkade and the docs, but `OPENFAAS_PWD` that you picked also can work.
